__tests__/app.test.js

```diff
@@ -114,6 +113,46 @@ describe('/api/reviews/:review_id', () => {
                     expect(body.msg).toBe("Bad request");
                 });
         });
+        test("should now show comment count = 0", () => {
```

Comment:
your above test description for invalid id should match your status code within your test. 400 not a 404